nodes.py

```diff
@@ -62,9 +62,12 @@ def INPUT_TYPES(s):
 
     def encode(self, clip, text):
         tokens = clip.tokenize(text)
+        if clip.use_clip_schedule:
```

Comment:
any reason this can't go in clip.encode_from_tokens instead of modifying every clip encode node?

Reply:
encode_from_tokens_scheduled will return the full list of all necessary conds to match the clip schedule, in the format [[cond, output]] - each [cond, output] corresponds to the set keyframes. The encode_from_tokens function is only expected to return stuff for just one cond (and not always the dictionary), so unfortunately it cannot be backported without an extra flag.

However, all the calls in these nodes to clip,encode_from_tokens could be replaced with clip.encode_from_tokens_scheduled only, so that encode_from_tokens would still function normally everywhere else.

Reply:
Relevant code is now cleaned up by just using the clip.encode_from_tokens_scheduled call.